From dcc6dae804a1b9c43540ccc1c34ee59e71098b08 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Mon, 30 Jul 2018 14:06:42 +0200
Subject: [PATCH] fix formatting

---
 reference_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/reference_test.go b/reference_test.go
index 4f6e773..7c70f64 100644
--- a/reference_test.go
+++ b/reference_test.go
@@ -266,7 +266,7 @@ func TestFileScheme(t *testing.T) {
 	}
 
 	if r1.IsCanonical() != true {
-		t.Errorf("NewJsonReference(%v)::IsCanonical %v expect %v", in1, r1.IsCanonical, true)
+		t.Errorf("NewJsonReference(%v)::IsCanonical %v expect %v", in1, r1.IsCanonical(), true)
 	}
 
 	result, err := r1.Inherits(r2)
-- 
2.7.5

